runtime/include/mthdecls.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#if !defined(_WIN64) && !defined(HOST_MINGW)
```

Comment:
What will happen in Windows?

Reply:
The macros are defined below.

Reply:
Ahh sorry, I missed the else in between.